GhidraBuild/IDAPro/Python/9xx/plugins/xml_importer.py

```
@@ -0,0 +1,99 @@
+#---------------------------------------------------------------------
+# xmlimp.py - IDA XML Importer plugin
+#---------------------------------------------------------------------
+"""
+Plugin for IDA to import a XML PROGRAM file into an existing open database.
+This file must be placed in the IDA plugins directory.
+The file idaxml.py must be placed in the IDA python directory.
+"""
+
+from __future__ import print_function
+import ida_idaapi
+import ida_pro
```

Ruff (0.12.2)

`ida_pro` imported but unused

Remove unused import: `ida_pro`

(F401)